apps/website/src/server/db/show-and-tell.ts

```diff
@@ -149,6 +150,7 @@ const showAndTellSharedInputSchema = z.object({
   location: z.string().max(MYSQL_MAX_VARCHAR_LENGTH).nullable(),
   longitude: z.number().nullable(),
   latitude: z.number().nullable(),
+  dominantColor: z.string().min(5).max(11).nullable(),
```

Comment:
Worth using a pattern here to ensure we have a comma-separated value of digits and not some arbitrary other string?